_includes/releases/v23.1/v23.1.0-rc.2.md

```diff
@@ -0,0 +1,19 @@
+## v23.1.0-rc.2
+
+Release Date: May 4, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-0-rc-2-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug introduced in versions 22.1.19, 22.2.8, and pre-release versions of 23.1 that could cause queries to return spurious insufficient privilege errors. For the bug to occur, two databases would need to have duplicate tables, each with a foreign key reference to another table. The error would then occur if the same SQL string was executed against both databases concurrently by users that have privileges over only one of the tables. [#102651][#102651]
```

Comment:
Not sure, what do you think about including these links:

[privilege](security-reference/authorization.html)
[foreign key](foreign-key.html)

Reply:
Done!